pkg/services/cloudmigration/model.go

```diff
@@ -254,3 +254,13 @@ const (
 	SnapshotStateError       SnapshotState = "ERROR"
 	SnapshotStateUnknown     SnapshotState = "UNKNOWN"
 )
+
+var (
+	ErrTokenInvalid           = errutil.Internal("cloudmigrations.createMigration.tokenInvalid", errutil.WithPublicMessage("Token is not valid. Generate a new token on your cloud instance and try again."))
```

Comment:
Do we need these `errutil.WithPublicMessage` since it is now being handled by the FE? Without it, I get this payload in the response:
```
{
    "message": "InternalError",
    "messageId": "cloudmigrations.createMigration.tokenInvalid",
    "statusCode": 500,
    "traceID": ""
}
```
So `getTMessage` still works since it uses the `messageId`, and `maybeAPIError` still returns an object since it has a `message`.

I am just thinking that it will be some work to keep those message in sync.

Reply:
Sounds good then!